echopype/convert/set_groups_ek80.py

```diff
@@ -71,6 +71,31 @@ def __init__(self, *args, **kwargs):
             for k, v in self.parsed2zarr_obj.p2z_ch_ids.items():
                 self.parser_obj.ch_ids[k] = self._get_channel_ids(v)
 
+        # construct sorted channel ids (ascending) for various usage scenarios
+        all_chan_ids = list(self.parser_obj.config_datagram["configuration"].keys())
+        all_chan_ids.sort(reverse=False)
+
+        power_chan_ids = self.parser_obj.ch_ids["power"]
+        power_chan_ids.sort(reverse=False)
+
+        complex_chan_ids = self.parser_obj.ch_ids["complex"]
+        complex_chan_ids.sort(reverse=False)
+
+        pow_comp_chan_ids = self.parser_obj.ch_ids["power"] + self.parser_obj.ch_ids["complex"]
+        pow_comp_chan_ids.sort(reverse=False)
+
+        angle_chan_ids = self.parser_obj.ch_ids["angle"]
+        angle_chan_ids.sort(reverse=False)
+
+        # obtain sorted channel dict in ascending order for each usage scenario
+        self.sorted_channel = {
+            "all": all_chan_ids,
+            "power": power_chan_ids,
+            "complex": complex_chan_ids,
+            "power_complex": pow_comp_chan_ids,
+            "angle": angle_chan_ids,
+        }
```

Comment:
This looks fine. I would suggest a small function
```
def sort_list(list_in):
   list_in.sort(reverse=False)
```
and then do something like:
```
self.sorted_channel = {
            "all": sort_list(list(self.parser_obj.config_datagram["configuration"].keys())),
            "power": self.parser_obj.ch_ids["power"],
            ... 
        }
```